### PR TITLE
missing test for searchable collection component

### PR DESCRIPTION
**Pull request by alexbowen**
no ticket, the spec for this component hadnt been added

### CODE REVIEW

**Comment by martin-bangoura** [comment]
Review app deployed to <https://teaching-vacancies-review-pr-2838.london.cloudapps.digital>

File: spec/components/shared/searchable_collection_component_spec.rb [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  let(:collection) do
+    [1, 2, 3, 4, 5].freeze
+  end
+
+  before do
+    allow(form).to receive(:govuk_collection_radio_buttons)
+    allow(form).to receive(:govuk_collection_check_boxes)
+  end
+
+  context "Radio button searchable collection" do
```
Comment (david-mears-dfe) [inline spec/components/shared/searchable_collection_component_spec.rb:15]: Our convention is that we either use `context "when(/with/without) something is the case"`, or `describe "feature/object/method"`. This comes from https://www.betterspecs.org/

File: spec/components/shared/searchable_collection_component_spec.rb [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                          collection: collection,
+                          text_method: :first,
+                          hint_method: :first,
+                          value_method: :first,
+                          threshold: 10,
+                          scrollable: true).with_variant(:radiobutton)
+    end
+
+    describe "renders radio button variant" do
+      let!(:inline_component) { render_inline(radio_collection) }
```
Comment (david-mears-dfe) [inline spec/components/shared/searchable_collection_component_spec.rb:28]: This line, and the duplicate of it, can be lifted up into the top `RSpec.describe` block. That then renders the `describe` lines 27 and 54 redundant.

**Review by david-mears-dfe — APPROVED** [review]
Pre-approved, but see comments.

File: spec/components/shared/searchable_collection_component_spec.rb [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+                          attribute_name: :attributes,
+                          collection: collection,
+                          text_method: :first,
+                          hint_method: :first,
+                          value_method: :first,
+                          threshold: 10,
+                          scrollable: true).with_variant(:radiobutton)
+    end
+
+    describe "renders radio button variant" do
```
Comment (cesidio) [inline spec/components/shared/searchable_collection_component_spec.rb:27]: As a convention we should use describe only for methods. In this case I would use 
`it "renders radio button variant" do`

**Pull request merged**